### ISSUE TITLE
Kuon BND Support

### ISSUE THREAD

**Issue by real-guilty** [ividyon/WitchyBND#40]
Wasn't clear if support for Kuon was added to Yabber/SoulsFormats, but Witchy does not parse ALL.BND. Posting this inquiry to clarify if the intent in the past was to support it or not.

**Comment by WarpZephyr** [comment]
I think I may have added support for it in my fork of Yabber called YabberExtended,
One reason they haven't been messed with much is because they are from very old games the devs don't focus on much.
Reverse engineers like TKGP used to go around randomly hunting for things and would just happen to add basic support.

These are rather annoying BNDs. They are probably all considered the 1st main version of Binder, and often leave out parts. There are things like alignment size, file version, and so on. Depending on the game it seems FromSoftware just had their tools leave bits of info out. Without flags like later versions have, you have a play a guessing game of which fields were turned on. I remember games like Kuon being particularly annoying because some of the inner BNDs used different field settings.

**Comment by ividyon** [comment]
There have been no particular plans, it's more of a function of demand & ease of execution. If the code to parse those file formats exists or somebody contributes it, and a user desires it in Witchy, it can probably be added without too much headache. This is the first time anyone mentions Kuon to me, so I can take a look, and coordinate with WarpZephyr who's been all about these older FS titles.

**Comment by real-guilty** [comment]
> YabberExtended

Indeed, this is my first time hearing of YabberExtended and I just tested it on ALL.BND. It unpacked similarly, if not exactly, the same as Chroxx's BMS script.

I've been trying to view the .tex and .tm2 images with ConsoleTextureExplorer and they both seemingly load with different configurations. I can't quite ascertain if the .tex is swizzled and  .tm2 is unswizzled as I've only tested the first few .tex files and the font .tm2 files.
